sdk/src/androidTest/java/com/bugsnag/android/ClientTest.java

```diff
@@ -303,28 +303,17 @@ public void testBreadcrumbStoreNotModified() {
     }
 
     @Test
-    public void testAppData() {
+    public void testAppDataCollection() {
         Client client = generateClient();
         AppData appData = client.getAppData();
-        assertNotNull(appData);
         assertEquals(client.getAppData(), appData);
     }
 
-    @Test
-    public void testAppDataSummary() {
-        Client client = generateClient();
-        AppData appData = client.getAppData();
-
-        Map<String, Object> appDataSummary = appData.getAppDataSummary();
-        assertNotNull(appDataSummary);
-        assertNotEquals(appData.getAppDataSummary(), appData);
-    }
-
     @Test
     public void testAppDataMetaData() {
         Client client = generateClient();
         Map<String, Object> app = client.getAppData().getAppDataMetaData();
-        assertEquals(5, app.size());
+        assertEquals(6, app.size());
```

Comment:
What is the 6th item? Should there be an assertion for that?

Reply:
The `codeBundleId` (which is null by default). I can add an assertion for this.

Reply:
I was thinking of the wrong object - it's actually `activeScreen`, which will be null in the unit test. I've added a check to ensure the key is present in the map.